docs/source/dpo_trainer.mdx

```diff
@@ -169,59 +130,71 @@ While training and evaluating we record the following reward metrics:
+| `"robust"`                             | The [Robust DPO](https://huggingface.co/papers/2403.00409) authors propose an unbiased estimate of the DPO loss that is robust to preference noise in the data. Like in cDPO, it assumes that the preference labels are noisy with some probability. In this approach, the `label_smoothing` parameter in the [`DPOConfig`] is used to model the probability of existing label noise. To apply this conservative loss, set `label_smoothing` to a value greater than 0.0 (between 0.0 and 0.5; the default is 0.0)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| `"bco_pair"`                           | The [BCO](https://huggingface.co/papers/2404.04656) authors train a binary classifier whose logit serves as a reward so that the classifier maps {prompt, chosen completion} pairs to 1 and {prompt, rejected completion} pairs to 0.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
shall we also say that the actual BCO Trainer is in the `BCOTrainer`?

Reply:
Good point. Are they equivalent? Why do we have both?

Reply:
so the one in the DPO is the "paired" version of BCO... in theory it can be removed... BCO is like KTO to a certain extent

Reply:
How would you formulate it for the doc?